.github/release.yml

```diff
@@ -0,0 +1,30 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+      - tests-only
+  categories:
+    - title:  ':tada: Exciting New Features'
+      labels:
+        - '*'
+      exclude:
+        authors:
+          - renovate[bot]
+    - title: 🐛 Bug Fixes
+      labels:
+        - bug
+    - title: ':mega: Community Contributions Highlight'
+      labels:
+        - community-pr
+    - title: ⚡ Performance
+      labels:
+        - performance
+    - title: 👒 Dependencies & Housekeeping
+      authors:
+        - renovate[bot]
```

Comment:
⚠️ From what I can see in the [docs](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuration-options) `authors` is only available in `exclude` section =>  `changelog.categories[*].exclude.authors`; so I think this config won't work or do anything.
Additionally, maybe we could have PRs created by `renovate[bot]` automatically have one of the labels listed below so to be added to this category.